values.yaml

```diff
@@ -648,35 +895,49 @@ syncCatalog:
   # True if you want to enable the catalog sync. Set to "-" to inherit from
   # global.enabled.
   enabled: false
+
+  # The name of the Docker image (including any tag) for consul-k8s
+  # to run the sync program.
+  # @type: string
   image: null
-  default: true # true will sync by default, otherwise requires annotation
+
+  # If true, all valid services in K8S are
+  # synced by default. If false, the service must be annotated
+  # (https://consul.io/docs/k8s/service-sync#sync-enable-disable) properly to sync.
+  # In either case an annotation can override the default.
+  default: true
 
   # Optional priorityClassName.
   priorityClassName: ""
 
   # toConsul and toK8S control whether syncing is enabled to Consul or K8S
   # as a destination. If both of these are disabled, the sync will do nothing.
   toConsul: true
+
+  # If true, will sync Consul services to Kubernetes. This can be disabled to
+  # have a one-way sync.
   toK8S: true
 
   # k8sPrefix is the service prefix to prepend to services before registering
```

Comment:
From here on down I see quite a few old style comments starting with the yaml key, just wanted to point out in case you wanted to fix those as well